HealthcareSystem/GraphicalEditor/Models/MapObjectRelated/MapObjectType.cs

```diff
@@ -12,11 +12,15 @@ public class MapObjectType
     {
         private MapObjectTypes _mapObjectType;
 
+        public MapObjectType()
+        {
+            this._mapObjectType = MapObjectTypes.PARKING;
+        }
         public MapObjectType(MapObjectTypes mapObjectType)
         {
             this._mapObjectType = mapObjectType;
         }
-
+        public MapObjectTypes GetMapObjectTypes { get => _mapObjectType; set => _mapObjectType = value; }
```

Comment:
Property names shouldn't contain 'get' or similar words. It's redundant at best and misleading at worst (like in this example, the name contains the word 'get' but the property has a setter as well - so someone could put 'object.GetMapObjectTypes = ...' somewhere in the code, which would be rather confusing).